unspun/FreeRTOS-Kernel
# [no upstream] Add FreeRTOS CMake support

# Add FreeRTOS CMake support

Description
-----------
This change adds a root CMakeLists.txt file to build FreeRTOS for GCC/ARM_CM4F. See README-unspun.md for usage. 

Test Steps
-----------
1. Added a test project that includes and builds FreeRTOS as a library
2. Added a github action to confirm the build works
